sql/7_0_0-to-7_0_1_upgrade.sql

```diff
@@ -279,7 +279,7 @@ UPDATE `registry` SET `category` = 'Orders' WHERE `directory` = 'procedure_order
 ALTER TABLE `insurance_data` ADD `date_end` date NULL;
 #EndIf
 
-#IfNotColumnType user user VARCHAR(255)
+#IfNotColumnType form_questionnaire_assessments user VARCHAR(255)
```

Comment:
sorry!:(

Reply:
no prob. i should of caught that in the review :) (btw, will also bring this in to patch.sql script in rel-700)